### PR TITLE
Nigel Douglas PCI DSS blog

### PR DESCRIPTION
**Pull request by nigeldouglas-itcarlow**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind user-interface

 /kind content

> /kind translation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

 /area blog

> /area documentation

> /area videos

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: content/en/blog/falco-pci-dss/index.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+        Exceptions are made for known trusted images.
+  condition: >
+    container_started and container
+    and container.privileged=true
+    and not falco_privileged_containers
+    and not user_privileged_containers
+    and not redhat_image
+  output: Privileged container started (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline)
+  priority: INFO
+  tags: [container, privilege_escalation, lateral_movement, T1610, PCI_DSS_10.2.5]
```
Comment (incertum) [inline content/en/blog/falco-pci-dss/index.md:67]: @nigeldouglas-itcarlow oh I love this convention about the PCI/DSS reference, can we learn more it, what standards did you base it on (I am more of a threat detection and less compliance person). We are discussing a new rules maturity and adoption framework here https://github.com/falcosecurity/rules/pull/76 and we wanted to add at least 3 "compliance" rules by Falco 0.36 and/or tag existing ones in this regard? Could you help us? Any feedback welcome re the rules maturity framework proposal!

**Review by Issif — APPROVED** [review]

File: content/en/blog/falco-pci-dss/index.md (hunk 1 above)
Comment (n1g3ld0ugla5) [reply content/en/blog/falco-pci-dss/index.md:67]: @incertum I am happy to help with this. I only tagged the existing rules with the relevant PCI DSS tags based on the standards/controls outlined in the PCI compliance framework - https://www.pcidssguide.com/pci-dss-requirement-10/

I am also working on aligning some existing Falco rules with NIST 800-171 Controls. If this helps?
Either way, I'd be happy to discuss this offline. Is there a meeting set up for the maturity framework proposal?

**Pull request merged**